.github/workflows/update-wiki.yml

```diff
@@ -0,0 +1,59 @@
+name: Update JSON files on wiki
+
+on:
+  push:
+    branches: [ master, jsondump ]
+    paths:
+      - '**.cs'
+      - '**.csproj'
+      - '**.sln'
+      - '**.git**'
+      - '**.yml'
+      - 'RobustToolbox/'
+
```

Comment:
imo this should probably only push on chem-related changes. but maybe that is hard to specify

Reply:
At the very least it doesn't cost us anything if it pushes too frequently.

Reply:
This has been updated to a narrower list of globs, resolving.